curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-json-with-the-javascript-fetch-method.md

```diff
@@ -0,0 +1,165 @@
+---
+id: 5ccfad82bb2dc6c965a848e5
+title: Get JSON with the JavaScript fetch method
+challengeType: 6
+---
+
+## Description
+<section id='description'>
+Another way to request external data is to use the JavaScript <code>fetch()</code>  method. It is equivalent to XMLHttpRequest, but the syntax is considered easier to understand.
+Here is the code for making a GET request to <code>/json/cats.json</code>
+
+```js
+
+fetch('/json/cats.json')
+	.then(response => response.json())
+	.then(data => {
+		document.getElementById('message').innerHTML=JSON.stringify(data);
+	})
+
+```
+Let's analyze each piece of this code. 
+
+The first line is the one that makes the request. So, <code>fetch(URL)</code> makes a GET request to the URL specified. The method returns a Promise.
+
+After a Promise is returned, if the request was successful, the <code>then</code> method is executed, which takes the response and converts it to JSON format.
+
+The <code>then</code> method also returns a Promise, which is handled by the next <code>then</code> method. The argument in the second <code>then</code> is the JSON object you are looking for! 
+
+Now, you select the element you want to insert the data in by using <code>document.getElementById()</code>. Then you modify the HTML code of the element by inserting a string created from the JSON object returned from the request.
```

Comment:
```suggestion
Now, it selects the element that will receive the data by using <code>document.getElementById()</code>. Then it modifies the HTML code of the element by inserting a string created from the JSON object returned from the request.
```
I found the original sentence a bit confusing.  Also, it seemed better to stick with the idea that the code is doing the things, not you.